### PR TITLE
8438 visual feedback when hovering dropzone

### PR DESCRIPTION
**Pull request by rianfowler**
dragging

### CODE REVIEW

File: src/js/vic-v2/components/PhotoField.jsx [hunk 1]
```diff
@@ -733,8 +738,14 @@ export default class PhotoField extends React.Component {
           </div>
           }
           {fieldView === 'initial' && <div className="drop-target-container">
-            <Dropzone className="drop-target" onDrop={this.onChange} accept="image/jpeg, image/jpg, image/png, image/tiff, image/tif, image/bmp">
-              <img alt="placeholder" src="/img/photo-placeholder.png"/>
+            <Dropzone
+              className={`drop-target ${this.state.dragging && 'dragging'}`}
```
Comment (rianfowler) [inline src/js/vic-v2/components/PhotoField.jsx]: `Dropzone` has an `activeClass` prop but it only takes an inline style object vs. a className.

File: src/js/vic-v2/components/PhotoField.jsx [hunk 2]
```diff
@@ -733,8 +738,14 @@ export default class PhotoField extends React.Component {
...
+              onDragEnter={() => this.handleDrag(true)}
```
Comment (rianfowler) [inline src/js/vic-v2/components/PhotoField.jsx]: `handleDrag` is kind of a misnomer- open to suggestions.

File: src/js/vic-v2/components/PhotoField.jsx [hunk 3]
```diff
@@ -549,6 +550,10 @@ export default class PhotoField extends React.Component {
     }
   }
 
+  handleDrag(dragging) {
+    this.setState({ dragging });
```
Comment (rianfowler) [inline src/js/vic-v2/components/PhotoField.jsx]: dragging? isDragging?

File: src/js/vic-v2/components/PhotoField.jsx (hunk 2 above)
Comment (jbalboni) [reply src/js/vic-v2/components/PhotoField.jsx]: `setDragging`? Not really sure either.

**Comment by rianfowler** [comment]
I used DROP IMAGE but I just realize the nomenclature used in the initial view is PHOTO. Will update.

**Review by jbalboni — APPROVED** [review]

**Pull request merged**